canonical-web-and-design/snapcraft.io
# Refactor releases code into the release redux actions

Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/2275

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/snap_name/releases
- Ensure releasing revisions works as expected
